### DISCUSSION TITLE
[米家同好群]招募

### DISCUSSION THREAD

**Discussion by shidijun** [share121/inter-knot#4612] [绝区零]
<div>
<div align="center"><h1>[米家同好群]招募</h1></div>
<p>
向各位米游玩家们致以敬意。
</p>
米游玩家一直希望有个好的圈内氛围已是游戏圈人尽皆知的事实。怎奈天违人愿，总有一些米哈游黑粉（以下简称“米黑”）打着以各种各样、虚实相掺的旗号，刻意扰乱米圈氛围和抹黑米游及米游玩家对外形象。对此，我们希望米游玩家团结起来，而不是被不良舆论影响，或是“胳膊肘往外拐”。
<p>
以召集米游玩家、创建一个良好氛围的圈子供各位米游玩家进行友好讨论的目的，我们一众志同道合的玩家创建了一个微信群聊，恭迎各位米游玩家通过添加群主微信加入（群主微信见“附2”）或是添加“史蒂君ovo”好友加入（见“附3”）（或可添加附4），无关人士还望止步。
</p>
<p>
本群聊为正经、合法群聊，望加入者遵守群公告中的各项规定，违者后果自负；无关人士一概不予入群，无关人士入群者核实后踢出群聊。
</p>
<p>
恭迎各位有意的米游玩家加入。
</p> 
<p></p>
<p></p>            
                                                         <div align="right"><p>米家同好群 （writer:by史蒂君）</p>                                 
                                                                        <p>2024.9.8</p></div>

<h2>附1</h2>
<img src="https://github.com/user-attachments/assets/94e332be-d5fd-4a6f-9c8c-eb2b32027fc2" alt="1">
<h2>附2</h2>
<img src="https://github.com/user-attachments/assets/11ea934f-bc2d-4cfa-98bd-d8f4e67fe202">
<h2>附3</h2>
<img src="https://github.com/user-attachments/assets/b59ec74f-9ba8-43a1-a075-9f901b4e3a4d" alt="2">
<h2>附4</h2>
<img src="https://github.com/user-attachments/assets/d9cc7391-b530-4455-a1b3-07e273ca038e" alt="3">

**Comment by shidijun** [comment]
作者来了 解决了一些问题<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >

**Comment by shidijun** [comment]
没热度 顶顶顶<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >

**Comment by omnisss** [comment]
来了来了qwq

**Comment by xiarideling** [comment]
蹭蹭<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/62158b26331a045bbaabf48d1fc5c8eb_980401343708652819.png" alt="阿姬灵感" width="75" >

**Reply by shidijun** [reply]
抓到你啦<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/09/04/e769d98158b8ae97d55fdd9629373a2b_8560015538581788584.png" alt="玛拉妮-观察" width="75" >让我康康

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/09/04/a53d037101e8300130d934ed57ee6ef2_6558580448093536651.png" alt="玛拉妮-赢了" width="75" >表情包发送成功！

**Reply by shidijun** [reply]
抓到你啦

**Comment by shidijun** [comment]
没热度了 我顶顶顶

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/09/04/a7904cfbd304ec820f4bc4ab2b5d45a6_293682765658091496.png" alt="基尼奇-“伪装”" width="75" >让我看看怎么个事

**Comment by anewperson001** [comment]
别看啦，进来吧你！<img src="https://bbs-static.miyoushe.com/static/2024/09/04/b3ee443dbdb192159e12416632f822cb_60085526798951291.png" alt="基尼奇-赶路" width="75" >

**Comment by YOLOjj** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
试试表情包

**Reply by anewperson001** [reply]
<img src="https://img-static.mihoyo.com/communityweb/upload/c6a13a33cc28af3f0dadac0631b07da0.png" alt="好顶赞" width="75" >加油加油

**Comment by anewperson001** [comment]
当当~打卡成功<img src="https://img-static.mihoyo.com/communityweb/upload/951272c9ce086ecb1baefd179f18973c.png" alt="庆贺吧" width="75" >

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/eb81560ecf3e308280f8d3599f24b64d_8586799033270022982.png" alt="丽都漫步-艾莲" width="75" >zzz，本人已不在营业期，zzz，勿cue，zzz

**Comment by Oumainory** [comment]


**Reply by anewperson001** [reply]
<img src="https://bbs-static.miyoushe.com/static/2024/09/04/f2ca6cb970baa36ebb1b8950f3894fb2_4590307880523876022.png" alt="玛拉妮-慌张" width="75" >暂时没有哦<img src="https://bbs-static.miyoushe.com/static/2024/07/31/83388b36f5f3e91b1c5815a39b89b15c_8086283022680108242.png" alt="云璃 好吃" width="75" >

**Reply by shidijun** [reply]
没有哦 群主没给意见呢<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3248c770e7e0e7ec4a167605872b6d66_1511725123905663480.png" alt="丽都漫步-格莉丝" width="75" >

**Comment by haoyunba** [comment]
点击[http://pinyin.cn/e368251]查看表情

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/09/04/a9fbeee1e21cf1abcf7bf2e610ee91f5_8437422338490524317.png" alt="玛拉妮-气氛组" width="75" >施工中~嘟嘟嘟——看，出货啦！<img src="https://bbs-static.miyoushe.com/static/2024/09/04/70420b7df8a3934cb8e7b0f0f4136511_7657550741734618773.png" alt="卡齐娜-囤货" width="75" >

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/05/13/daf22495e5ef8f11fd812684f641c10b_6950741619984342613.png" alt="阿蕾奇诺-合作愉快" width="75" >出货？呵呵~拿来吧你！

**Comment by anewperson001** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/6133bdabc2bb52bd5c504cecf57d380d.png" alt="温迪-撒花" width="75" >任务圆满完成，群里大水怪敬上~<img src="https://img-static.mihoyo.com/communityweb/upload/5dfae09fdbd9f647f57eeaeeda72f65b.png" alt="诺艾尔-超赞" width="75" >

**Comment by shidijun** [comment]
作者终于放学了 回来了 群里从50多人到了92人 谢谢大家支持ovo

**Comment by anewperson001** [comment]
欢迎新人，记得更新二维码<img src="https://bbs-static.miyoushe.com/static/2024/07/31/213614cd6275ec54d80fe9f84a0edf62_6334766189957723007.png" alt="白露 灭火" width="75" >

**Reply by shidijun** [reply]
好哒 <img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Comment by shidijun** [comment]
顶

**Comment by Minapocalis** [comment]
应该没问题吧

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/213614cd6275ec54d80fe9f84a0edf62_6334766189957723007.png" alt="白露 灭火" width="75" >

**Comment by shidijun** [comment]
顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶
<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >

**Comment by kinlaite** [comment]
顶顶顶

**Comment by shidijun** [comment]
顶顶顶<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3248c770e7e0e7ec4a167605872b6d66_1511725123905663480.png" alt="丽都漫步-格莉丝" width="75" >

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/02/02/2f87e1eec1b7390e66363e29c616d011_7092543438575032192.png" alt="希娜狄雅-不愧是我" width="75" >原来你就是群里经常提到的希娜啊？你好你好~

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/06/28/5d43ad69b0cc8e6914bca943e61c9ae0_684811288942066653.png" alt="瑟莉姆-哼哼" width="75" >嗯哼？<img src="https://bbs-static.miyoushe.com/static/2024/06/28/05095a32aec789ea7e7fe6f03d88b364_6022440254094262976.png" alt="瑟莉姆-闭嘴" width="75" >

**Reply by shidijun** [reply]
抓到啦<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >

**Comment by shidijun** [comment]
顶顶顶<img src="https://img-static.mihoyo.com/communityweb/upload/1a25d24ab81aaa88543ffad7cbf3f944.png" alt="绝区零_暗中观察" width="75" >
